@xen-orchestra/web-core/lib/components/CardNumber.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="card-number">
```

Comment:
To match the component name:
```suggestion
  <div class="card-numbers">
```